tests/cli/NewFwkLib/PdxTest/PdxTests.cs

```diff
@@ -231,7 +231,7 @@ protected TVal createIpdxInstance()
             pif.WriteByteArray("m_byte253", pt.Byte253);
             pif.WriteByteArray("m_byte65535", pt.Byte65535);
             pif.WriteByteArray("m_byte65536", pt.Byte65536);
-            pif.WriteObject("m_pdxEnum", pt.PdxEnum);
+            //pif.WriteObject("m_pdxEnum", pt.PdxEnum);
```

Comment:
just delete?

Reply:
Nope, need to add that back in. Good eye!